src/modules/server.ts

```diff
@@ -34,13 +35,15 @@ class Server {
 
     private async setRoutes() {
       this.router.get('/hello', hello);
+      this.router.get('/info', info);
     }
 
     public async startServer() : Promise<httpServer | void> {
       await asyncRetry(async (bail) => {
         try {
           await this.setPort();
           await this.setRoutes();
+          await Database.getSingleton();
```

Comment:
why you did this? Since this method return an object and you are not receiving it.

Reply:
Solved on last commit